com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -62,21 +67,25 @@ def __setattr__(self, key, value):
         super(_ResourceElement, self).__setattr__(key, value)
 
     def refresh(self):
-        """Make a REST call to refresh the JSON object stored within the object
-        Returns:
-            _ResourceElement: The provided object instance is returned with updated value
+        """Refresh the JSON representation of the resource for the latest status
```

Comment:
Why is it the "JSON represresentation"?

Isn't it just refreshing the object, the fact that JSON was used is an implementation detail isn't it? By the time the user gets the object it's not JSON, just a Python object.